action.yml

```diff
@@ -12,6 +12,12 @@ inputs:
 runs:
     using: "composite"
     steps:
+        - run: echo "::add-mask::$CONDUCTOR_TOKEN"
```

Comment:
This section is only there for the masking of the token further down, correct?

Unfortunately the token is still shown under `env`. I played around with this and I couldn't find any straightforward solution for this. Is this acceptable, considering that these are short-lived tokens?
![Screenshot 2025-02-18 at 01 07 36@2x](https://github.com/user-attachments/assets/b2fa3432-c957-4d17-a4ff-810742afab85)

Reply:
Ah, missed that the ENV is displayed as part of the command. Really wanted to find a way to no display this in the output. Let me give this another try.

Reply:
@IgorBenko found [a solution here](https://github.com/actions/runner/issues/643#issuecomment-823537871). Idea is parse `GITHUB_EVENT_PATH` with `jq`, this way the output looks like:

```
Run CONDUCTOR_TOKEN=$(jq -r '.client_payload.composerAuthentication.token' $GITHUB_EVENT_PATH)
  CONDUCTOR_TOKEN=$(jq -r '.client_payload.composerAuthentication.token' $GITHUB_EVENT_PATH)
  echo "::add-mask::$CONDUCTOR_TOKEN"
```

Reply:
Works perfectly now 💪